artacle-swagger.yaml

```diff
@@ -314,6 +314,281 @@ paths:
         '500':
           $ref: "#/components/responses/InternalServerError"
 
+#  Collection Traits
```

Comment:
General: as requested by Alex, let's specifiy somewhere (we can do it in Summary of each method or somehow before the whole Traits block): "Trait's TTF (Trait (sale) to Floor ratio) is calculated as the ratio of prices of the last sale of each token having specific trait to the collection's floor price at the moment of sale. Outliers get filtered out, median is used to provide aggregated TTF characteristic that can estimate the trait's value"

Reply:
added as tag description